src/interface/blas3/backend/intel_gpu.hpp

```diff
@@ -39,8 +39,8 @@ typename executor_t::policy_t::event_t _gemm(
     gemm_batch_type_t batch_type) {
   if (batch_type == gemm_batch_type_t::interleaved) {
     return blas::Gemm_Launcher<
-        64, false, false, false, 64, Tile<4, 4, 4, 4, 1, 1, 1, 1, 4, 4>, _t_a, _t_b,
-        static_cast<int>(gemm_memory_t::no_local),
+        64, false, false, false, 64, Tile<4, 4, 4, 4, 1, 1, 1, 1, 4, 4>, _t_a,
```

Comment:
Why has this formatting changed? The clang-format file hasn't been updated as far as I can see.